client/my-sites/stats/navigation-arrows/index.jsx

```diff
@@ -0,0 +1,42 @@
+import { Icon, arrowLeft, arrowRight } from '@wordpress/icons';
+import classNames from 'classnames';
+
+import './style.scss';
+
+function NavigationArrows( {
+	disableNextArrow,
+	disablePreviousArrow,
+	onArrowNext,
+	onArrowPrevious,
+} ) {
+	const handleClickNext = () => {
+		onArrowNext();
+	};
+	const handleClickPrevious = () => {
+		onArrowPrevious();
+	};
+	return (
+		<div className="stats-navigation-arrows">
+			<button
+				className={ classNames( 'stats-navigation-arrows__previous', {
+					'is-disabled': disablePreviousArrow,
+				} ) }
+				onClick={ disablePreviousArrow ? undefined : () => handleClickPrevious() }
```

Comment:
I'm just wondering why the functions are wrapper by arrow functions a second time here (firstly in *1), is it necessary? IMHO, the first wrapping (*1) is not necessary as well or am I missing something?

Reply:
You're right! I like to have a local handler for the purpose of logging (during development) or pre-processing. I should have caught that all the logic was removed and just used the callbacks from props. I'll get this fixed up!